File: BaiduMap/Request/PlaceDetailRequest.cs
using BaiduMap.Request.Models;
using BaiduMap.Response;
using System;
using System.Collections.Generic;
using System.Text;

namespace BaiduMap.Request
{
    public class PlaceDetailRequest : BaiduRequest<PlaceDetailResponse, PlaceDetailModel>
    {
        public PlaceDetailRequest(PlaceDetailModel model) : base(model)
        {
            this.Address = "/place/v2/detail";
        }
    }
}

File: JAVA/baidumap-webapi/src/main/java/wang/seamas/baidumap/request/PlaceDetailRequest.java
package wang.seamas.baidumap.request;

import wang.seamas.baidumap.request.model.PlaceDetailModel;
import wang.seamas.baidumap.response.PlaceDetailResponse;

/**
 * @author: Seamas Wang
 * @description:
 * @date: Created in 11:07 2018/8/6
 */
public class PlaceDetailRequest extends BaiduRequest<PlaceDetailResponse, PlaceDetailModel> {
    public PlaceDetailRequest(PlaceDetailModel model) {
        super(model);
        this.address = "/place/v2/detail";
    }
}
